pom.xml

```diff
@@ -24,6 +24,8 @@
 
   <properties>
     <findbugs.failOnViolation>true</findbugs.failOnViolation>
+    <checkstyle.failOnViolation>true</checkstyle.failOnViolation>
+    <checkstyle.logViolationsToConsole>true</checkstyle.logViolationsToConsole>
```

Comment:
FYI: As pointed out by Bojan today, this property has no effect: https://github.com/kiegroup/jbpm-wb/pull/807#discussion_r128999897